spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+    project_id { 1 }
+
+    trait 'completion_task' do
+      status { 'done' }
```

Comment:
enumの値はStringではなくSymbolの形式で記載した方が良いですね！
```suggestion
      status { :done }
```